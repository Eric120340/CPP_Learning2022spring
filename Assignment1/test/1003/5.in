4 3
76 40 51 
89 96 15 
98 12 98 
32 37 79 
3 9
77 92 73 57 44 25 85 4 57 
27 20 79 72 35 37 76 99 31 
5 74 71 55 15 18 21 64 82 
