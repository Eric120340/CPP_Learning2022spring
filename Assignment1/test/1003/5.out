7187 11566 12329 10017 5509 4298 10571 7528 9754 
9520 11218 15146 12810 7501 6047 15176 10820 9279 
8360 16508 15060 11840 6202 4658 11300 7852 13994 
3858 9530 10868 8833 3888 3591 7191 8847 9449 
